cypress/integration/learn/index.js

```diff
@@ -48,3 +48,159 @@ describe('Learn Landing page', function() {
     });
   });
 });
+
+describe('Quotes', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains("Get started (it's free)").click({ force: true });
+  });
+
+  it('Should show a quote', () => {
+    cy.get('blockquote').within(() => {
+      cy.get('q').should('be.visible');
+    });
+  });
+
+  it('Should show quote author', () => {
+    cy.get('blockquote').within(() => {
+      cy.get('cite').should('be.visible');
+    });
+  });
+});
+
+describe('Superblocks and Blocks', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains("Get started (it's free)").click({ force: true });
+  });
+
+  it('Has first superblock and block collapsed by default', () => {
+    cy.contains(superBlockNames[0]).should('be.visible');
+    cy.contains(superBlockNames[0]).should(
+      'have.attr',
+      'aria-expanded',
+      'true'
+    );
```

Comment:
```suggestion
    cy.contains(superBlockNames[0])
      .should('be.visible')
      .and('have.attr', 'aria-expanded', 'true');
```
We can save a few lines by chaining off `should` here and later on.